build.sbt

```diff
@@ -43,12 +43,39 @@ lazy val units = crossProject(JVMPlatform, JSPlatform/*, NativePlatform*/)
   .dependsOn(core % "compile->compile;test->test")
   .settings(commonSettings :_*)
 
+lazy val testkit = crossProject(JVMPlatform, JSPlatform/*, NativePlatform*/)
+  .crossType(CrossType.Pure)
+  .in(file("testkit"))
+  .settings(
+    name := "coulomb-testkit",
+    libraryDependencies ++= Seq(
+      "org.scalacheck" %%% "scalacheck" % "1.15.4",
+      "org.scalameta" %%% "munit-scalacheck" % "0.7.29" % Test,
+    )
+  )
+  .dependsOn(core % "compile->compile;test->test")
+  .settings(commonSettings :_*)
```

Comment:
@erikerlandson would you be open to accepting this `testkit` module? Even if you don't test laws internally we'd find it helpful in our projects.

Reply:
sure, law checking is good CI